### DISCUSSION TITLE
Would it be possible to save Image Prompt (text / file path) data in the log file, as like generation parameters?

### DISCUSSION THREAD

**Discussion by Norby123** [lllyasviel/Fooocus#2107] [Ideas]
So, I absolutely love using the Image Prompt feature, for me this is the best way to get the result I am actually looking for.

But the log file doesn't save what reference images I used for certain generations.

Would it be possible to save the file path of those reference images that were used in the Image Prompt option (along with the stop/weight settings) in the log file? Kinda how the generated images are shown in the log file, where it basically just points to the file of the generated image on your disk. 
It doesn't even have to display the reference images used in Image Prompt, just have their file location path embedded in there, so if I want to recreate that generated image, I'll know what reference images I used in the Image Prompt.

This (maybe) could mess up the clipboard copy-paste generation data text thing, but if the reference images haven't been moved, then it should be alright, I guess?

Thanks in advance!
peace

**Comment by mashb1t** [comment]
I'm currently working on exactly this ^^
See https://github.com/lllyasviel/Fooocus/pull/1940, still WIP but ready in a few days.
